### ISSUE TITLE
3.4.0 Failure to keep utterance names the same in temporal partition and segment tier:  CN can reproduce

### ISSUE THREAD

**Issue by cneidle** [DCS-LCSR/SignStream3#625]
This should not be possible:

<img width="311" alt="Screen Shot 2023-01-24 at 11 24 39 AM" src="https://user-images.githubusercontent.com/13629122/214349921-074ca33f-f23a-4647-ac97-08297333c75c.png">

<img width="451" alt="Screen Shot 2023-01-24 at 11 24 20 AM" src="https://user-images.githubusercontent.com/13629122/214349916-beff06e4-6035-46b8-9be5-561aa11fc743.png">

I'm not sure how this happened.  I intended to type WORK_AGENT for the first and WRITE_SENTENCE for the second...

**Comment by gregorydimitriadis** [comment]
Yes - I'm also not sure how that would have happened, as there is no isolated way to rename an utterance from the segment tier row.

**Comment by cneidle** [comment]
This may have been related to the fact that I had renumbered the utterances, prior to renaming them.  I don't know.

**Comment by cneidle** [comment]
Here's one way to reproduce this.  I click on the "U21" label and I accidentally get brought into the utterance window, but the text box persists.  If I enter the text while in the utterance window and then go back to the database window, this is what I see:
<img width="310" alt="Screen Shot 2023-02-02 at 4 09 24 PM" src="https://user-images.githubusercontent.com/13629122/216450239-66b1c744-b162-4d9c-822e-4a9cccea7338.png">

**Comment by cneidle** [comment]
The persistence of the Utterance label box in the utterance window, mentioned just above, was shown in another ticket:

<img width="812" alt="Screen Shot 2023-02-02 at 4 14 28 PM" src="https://user-images.githubusercontent.com/13629122/216451057-9ba22d5e-0bb1-4f93-b154-8b3224a702f0.png">

Here's another example:

<img width="1037" alt="Screen Shot 2023-02-02 at 4 26 22 PM" src="https://user-images.githubusercontent.com/13629122/216453332-f2678619-058c-46d8-a9a0-5ce0562840c3.png">

**Comment by cneidle** [comment]
Another scenario:  I click on "U31".  The textbox from the previous utterance is activated. I click on U31 again, and the previous utterance name appears in the segment tier label for U31.

<img width="636" alt="Screen Shot 2023-02-02 at 4 18 18 PM" src="https://user-images.githubusercontent.com/13629122/216452028-6fce477c-b637-4aaf-8969-f1323769a7c6.png">

<img width="661" alt="Screen Shot 2023-02-02 at 4 18 28 PM" src="https://user-images.githubusercontent.com/13629122/216452047-ac4271f8-c1bd-4304-b4cf-7f7764e77510.png">

<img width="161" alt="Screen Shot 2023-02-02 at 4 18 36 PM" src="https://user-images.githubusercontent.com/13629122/216452066-60230f57-db2f-4fd0-aa5e-704d2794788b.png">

**Comment by cneidle** [comment]
Happened again.  While ROBOT was selected, I clicked on the following Utterance name; exact same sequence of events as above.

<img width="475" alt="Screen Shot 2023-02-02 at 4 21 58 PM" src="https://user-images.githubusercontent.com/13629122/216452599-d1c9c8bc-0701-42cd-9b65-483959795450.png">

<img width="428" alt="Screen Shot 2023-02-02 at 4 22 31 PM" src="https://user-images.githubusercontent.com/13629122/216452671-5e57756c-cb21-4362-ac86-cfaa08e6c340.png">

**Comment by cneidle** [comment]
Another sequence:

I click on SQUEEZE 

<img width="428" alt="Screen Shot 2023-02-02 at 4 29 21 PM" src="https://user-images.githubusercontent.com/13629122/216453924-a8191ffa-1461-4aee-a415-c483d068e31f.png">

I click on U39

<img width="485" alt="Screen Shot 2023-02-02 at 4 29 29 PM" src="https://user-images.githubusercontent.com/13629122/216453946-c15f4f9e-b41a-4210-8f3c-9ae1f1e29e8e.png">

I click on U39 again

<img width="442" alt="Screen Shot 2023-02-02 at 4 29 38 PM" src="https://user-images.githubusercontent.com/13629122/216453968-2490dd84-065d-44bf-8135-50dab0b0004e.png">

I click on U39 again

<img width="438" alt="Screen Shot 2023-02-02 at 4 29 49 PM" src="https://user-images.githubusercontent.com/13629122/216453992-f3635d0f-d63e-4c88-a560-f0f2d4e5626e.png">

**Comment by cneidle** [comment]
I opened RIT-SignBank-2a.ssc and found this!   Obviously should not be possible:

<img width="566" alt="Screen Shot 2023-02-02 at 9 44 01 PM" src="https://user-images.githubusercontent.com/13629122/216500328-bdb4fdb3-7e4b-4aba-aa7f-c58e81f929aa.png">

**Comment by cneidle** [comment]
When I tried to add an utterance to the temporal partition above, this happened:

<img width="502" alt="Screen Shot 2023-02-02 at 9 46 22 PM" src="https://user-images.githubusercontent.com/13629122/216500508-2aec7b76-7610-42a8-927d-04b9ab6792e5.png">

**Comment by cneidle** [comment]
I can delete the "U50" above, but I have no way to delete NORTH_WEST on the segment tier.

<img width="458" alt="Screen Shot 2023-02-02 at 9 48 13 PM" src="https://user-images.githubusercontent.com/13629122/216500700-45943e60-71b1-46e6-9e6b-6e5b4f53361d.png">

**Comment by cneidle** [comment]
I should note that I accidentally labelled two utterances in a row with the same name.
There are two occurrences of "NORTH+WEST" in a row.  I meant to add an extra _ 
in the second one, but neglected to do so.  I deleted the second one, but I still
can't delete that first one...

<img width="473" alt="Screen Shot 2023-02-02 at 9 50 41 PM" src="https://user-images.githubusercontent.com/13629122/216501069-77eeb314-7836-4ff6-8c73-dfaf807801e1.png">

**Comment by cneidle** [comment]
I then saved the file with the second one deleted and re-opened it, and things appeared as they should.

<img width="253" alt="Screen Shot 2023-02-02 at 9 52 53 PM" src="https://user-images.githubusercontent.com/13629122/216501334-c620b6f0-df5e-47fa-b76b-4669bb899de1.png">

**Comment by cneidle** [comment]
related to #624
